src/server/acl/acl_family.cc

```diff
@@ -419,6 +419,42 @@ void AclFamily::Cat(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void AclFamily::GetUser(CmdArgList args, ConnectionContext* cntx) {
+  auto username = facade::ToSV(args[0]);
```

Comment:
I've seen this in the last PR as well but didn't comment, we have the ArgS helper for that (its a little awkward but its used)

Reply:
Too lazy to make the change atm, will refactor it in some of the polishing PR's.